angular-ngrx-scss/src/app/issues/components/issues-list/issues-list.component.scss

```diff
@@ -1,3 +0,0 @@
-.issue {
```

Comment:
suggestion: let's delete this file if it'll stay empty! Can always add it back if it turns out we need it. :)